src/main/java/com/akademia/educators_back/valicators/ProblemValidator.java

```diff
@@ -0,0 +1,50 @@
+package com.akademia.educators_back.valicators;
+
+import com.akademia.educators_back.exception.CategoryDoesNotExistException;
+import com.akademia.educators_back.exception.ProblemAlreadyExistException;
+import com.akademia.educators_back.exception.ProblemDoesNotExistException;
+import com.akademia.educators_back.exception.TextIsToLongOrToShortException;
+import com.akademia.educators_back.mapper.ProblemMapper;
+import com.akademia.educators_back.repository.ProblemRepository;
+import com.akademia.educators_back.to.ProblemTo;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
nie ma konstruktora więc się wywali przy wywołaniu

Reply:
klasa wygląda dobrze i jest czytelna, z jakiś niuansów to komunikat z treścią błędu bym wywalił do jakiegoś zbiorczego enuma ale nie musisz tego zmieniać, będziemy jeszcze sobie robić refactoring kodu to wtedy zrobię zadanie żeby to zrobić dla całej apki, żeby było wszędzie jednolicie